src/cmd/src/cli/export.rs

```diff
@@ -409,14 +426,102 @@ impl Export {
 
         Ok(())
     }
+
+    async fn export_database_data(&self) -> Result<()> {
+        let timer = Instant::now();
+        let semaphore = Arc::new(Semaphore::new(self.parallelism));
+        let db_names = self.iter_db_names().await?;
+        let db_count = db_names.len();
+        let mut tasks = Vec::with_capacity(db_names.len());
+        for (catalog, schema) in db_names {
+            let semaphore_moved = semaphore.clone();
+            tasks.push(async move {
+                let _permit = semaphore_moved.acquire().await.unwrap();
+                tokio::fs::create_dir_all(&self.output_dir)
+                    .await
+                    .context(FileIoSnafu)?;
+                let output_dir = Path::new(&self.output_dir).join(format!("{catalog}-{schema}/"));
```

Comment:
`{catalog}-{schema}` maybe not a good choice. What about organizing the exported table like this:

```
catalog/
     table1/
     table2/
     table2/
```
It's more structural, and avoids the special characters in catalog or table names such as `-`.

Reply:
Do you mean this?

```
catalog/
     database1/
     database2/
     database2/
```

The current is:

```
     catalog-database1/
     catalog-database2/
     catalog-database2/
```

The smallest export unit is the database, not the table.

Reply:
We don't support exporting multiple `catalogs` at once, and `catalog` and `schema` names can't contain `-`, so an additional directory level isn't necessary. I think it's fine as is.

Reply:
But if the catalog name or database name contains some special characters, there will be some issues.Yes, i meant:

```
catalog/
     database1/
     database2/
     database2/
```

Reply:
Ok, I'll make the change.

Reply:
<img width="939" alt="image" src="https://github.com/user-attachments/assets/72d3a7c2-2cbd-477f-a242-142c0b600b2b">